docker/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM php:8.1.12-cli
+ARG COMPOSER_AUTH
+ENV COMPOSER_AUTH=$COMPOSER_AUTH
+
+RUN apt-get update && apt-get install -y -q git rake ruby-ronn zlib1g-dev && apt-get clean
```

Comment:
What is using rake and ronn in a PHP project?

Reply:
It looks like that step is needed to successfully install grpc.